libs/vkd3d/command.c

```diff
@@ -4803,6 +4803,18 @@ static HRESULT STDMETHODCALLTYPE d3d12_command_list_Close(d3d12_command_list_ifa
     /* If there are pending subresource updates, execute them now that all other operations have completed */
     d3d12_command_list_flush_subresource_updates(list);
 
+    if (list->has_unsynced_accel_struct_build)
```

Comment:
I'm not comfortable adding a hack like this unconditionally. If anything, has_unsynced_accel_struct_build should be in an anonymous "workaround" struct, to make it clear what the intention is. It should also be gated behind a vkd3d_config if anything.